admin/listPurchases/listView.html

```diff
@@ -52,24 +52,25 @@
     {
         cls: "option-all-orders",
         el: document.querySelector(".option-all-orders"),
-        filter: function(x) { return true; }
+        category: ["completed", "dispatched"]
     },
     {
         cls: "option-new-orders",
         el: document.querySelector(".option-new-orders"),
-        filter: function(x) { return x.status == "pending"; }
+        category: ["completed"]
```

Comment:
originally it was a filter applied to a changed version of the data
from listOrdersJson:
![image](https://github.com/user-attachments/assets/f0521311-8058-4fe9-bba4-bb2e46b34cdc)
completed is the status it has in the database which gets translated to pending in the frontend to avoid confusion of what completed means.